zerver/migrations/0014_realmemoji_name_regex.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import django.core.validators
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('zerver', '0013_merge'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='realmemoji',
+            name='name',
+            field=models.TextField(validators=[django.core.validators.MinLengthValidator(1), django.core.validators.RegexValidator(regex=b'^[0-9a-zA-Z_\\-\\.]*(?<![.\\-_])$')]),
+        ),
+    ]
```

Comment:
I think for these migrations it'd probably be cleaner to just have a single file 0013_realmemoji.py depending on 0012 and containing the code for the 0010 + 0014 added in this PR; there aren't many other database migration commits in flight.  Probably easiest way to do this would be to just delete these migrations and rerun makemigrations.